src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -129,6 +133,23 @@ class Hierarchy extends Extension
      */
     private static $prepopulate_numchildren_cache = true;
 
+    /**
+     * The default method called on the Hierarchy class to get children
```

Comment:
```suggestion
     * The default method called on the Hierarchy class to get children
     * The value of this configuration should be set on the base data class.
```

Or "base hierarchical class according to `getBaseHierarchyClass()`" or something along those lines if that's what you go for.

Any discussion about if using the base class is correct can be held in https://github.com/silverstripe/silverstripe-cms/pull/3077/files#r2087676209

Reply:
Have written an expanded docblock